src/fides/api/schemas/privacy_notice.py

```diff
@@ -145,37 +153,6 @@ class PrivacyNoticeResponse(PrivacyNoticeWithId):
     systems_applicable: bool = False
 
 
-class UserSpecificConsentDetails(FidesSchema):
-    """Schema for surfacing previously-saved preferences or previously-served
-    consent components
-    """
-
-    default_preference: Optional[
-        UserConsentPreference
-    ]  # The default preference for this notice or TCF component
-    current_preference: Optional[
-        UserConsentPreference
-    ]  # The current saved preference for the given user if it exists
-    outdated_preference: Optional[
-        UserConsentPreference
-    ]  # If no current preference, check if we have a preference saved for a previous version.
-    current_served: Optional[
-        bool
-    ]  # Do we have a record of the most recent version of this notice or TCF component being served to the user?
-    outdated_served: Optional[
-        bool
-    ]  # Have we served an older version of this notice or TCF component to the user?
```

Comment:
Obsolete - Plus should now use PrivacyNoticeResponse when it adds Privacy Notices to the Experience, not PrivacyNoticeResponseWithUserPreferences.

Reply:
Exactly yes